src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  text-transform: uppercase;
+}
+
+body {
+  margin: 0;
+  --blu: #00acdc;
```

move variables int `:root` pseudo selector
```suggestion
  --blue: #00acdc;
```